packages/nitro-protocol/test/contracts/ForceMove/challenge.test.ts

```diff
@@ -180,7 +181,7 @@ describe('challenge', () => {
 
       switch (challengeSignatureType) {
         case 'incorrect':
-          challengeSignature = {...correctChallengeSignature, v: correctChallengeSignature.v + 1};
```

Comment:
This (very hacky) approach was sometimes generating a valid but incorrect signature (desired), and sometimes generating an entirely invalid signature (undesired)